project/plugins.sbt

```diff
@@ -4,4 +4,6 @@ addSbtPlugin("org.scala-js" % "sbt-scalajs" % "0.6.33")
 
 addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "1.0.0")
 
-addSbtPlugin("com.codecommit" % "sbt-github-packages" % "0.5.3")
\ No newline at end of file
+addSbtPlugin("com.codecommit" % "sbt-github-packages" % "0.5.3")
+
+addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.9.11")
```

Comment:
It's `addSbtPlugin("com.github.sbt" % "sbt-native-packager" % "1.9.11")`

The namespace is incorrect. Can you push a PR on the `sbt-native-packager` to fix the doc, please?

Reply:
> Can you push a PR on the `sbt-native-packager` repo to fix the doc, please?

I doubt I am able to do that, as I have no idea where the documentation sources are located.

Reply:
Hum. Apparently it's fixed in the latest version of the doc: https://sbt-native-packager.readthedocs.io/en/latest/index.html

Reply:
Unfortunately the one I have coped the code from (https://www.scala-sbt.org/sbt-native-packager/) is higher ranked in Google results

Reply:
FYI https://users.scala-lang.org/t/https-www-scala-sbt-org-sbt-native-packager-is-oudated/8902

Reply:
Thanks.